src/client/components/General/Tabs/__tests__/Tabs.component.test.js

```diff
@@ -4,20 +4,21 @@
 
 import React from 'react';
 import sd from 'skin-deep';
+import {shallow} from 'enzyme';
```

Comment:
Det er rigtig fint at der er skiftet til enzyme, hvor meget arbejde vil det kræve at skrive skin-deep helt ud, så vi kan fjerne en dependency?

Reply:
skin-deep's shallowRender bliver brugt ca. 65 steder i tests og mit bud er at det tager ca. 6 timer at skrive det om til at bruge enzyme's shallow render.

Reply:
Jeg tænker at det nok ikke er det værd i denne omgang. Vi skal selvfølgelig bare sørge for at skifte det ud hen af vejen.

Reply:
👍